sqldelight-runtime/src/commonMain/kotlin/com/squareup/sqldelight/db/SqlDatabase.kt

```diff
@@ -17,13 +17,20 @@ package com.squareup.sqldelight.db
 
 import com.squareup.sqldelight.Transacter
 
+/**
+ * Maintains connections to an underlying SQL database and provides apis for managing transactions
+ * and executing SQL statements.
+ */
 interface SqlDatabase : Closeable {
   /**
-   * Prepare [sql] into a bindable object to be executed later.
+   * Execute a SQL statement and return a [SqlCursor] that iterates the result set.
    *
-   * [identifier] can be used to implement any driver-side caching of prepared statements.
+   * @param [identifier] Can be used to implement any driver-side caching of prepared statements.
```

Comment:
This describes how it can be used not what it is. Maybe:

An opaque, unique value that can...